pkg/server/config/config.go

```diff
@@ -81,5 +90,19 @@ func validate(config *CycloneServerConfig) bool {
 		config.CycloneServerPort = 7099
 	}
 
-	return true
+	if config.DefaultPVCConfig.Size == "" {
+		log.Warningf("DefaultPVCConfig.Size not configured, will use default value '5Gi'")
+		config.DefaultPVCConfig.Size = common.DefaultPVCSize
+	}
+
+	if config.WorkerNamespaceQuota == nil {
+		log.Warningf("WorkerNamespaceQuota not configured, will use default value limits.cpu limits.memory requests.cpu requests.memory")
```

Comment:
What about `WorkerNamespaceQuota not configured, will use default quota`?

Reply:
Use `Warningln` instead of `Warningf`, as there is no format.

Reply:
Use `Warningln` instead of `Warningf`, as there is no format.

Reply:
Ok, i will use `Warningln`

Reply:
What do you mean `default quota`? empty?

Reply:
Ok, i will use `Warningln`

Reply:
> What do you mean `default quota`? empty?

Just change the log content to make it simpler.